.github/PULL_REQUEST_TEMPLATE/release-checklist.md

```diff
@@ -167,6 +167,13 @@ and the updated changelog:
 - [ ] Publish the pre-release to GitHub using "Publish Release"
 - [ ] Delete all the [draft releases from the list of releases](https://github.com/ZcashFoundation/zebra/releases)
 
+## Publish crates
+
+- [ ] Install `cargo-release` if needed: `cargo install cargo-release`
+- [ ] Check that you are logged into `cargo` as an owner of the crate or a member of the `owners` group
+- [ ] Do a dry-run: `cargo release --workspace -v`
+- [ ] Publish the crates in tree-order: `cargo release --workspace -v --execute`
```

Comment:
I think the release steps should replace most of the `fastmod` steps:
```suggestion
<details>

<summary>If you're publishing crates for the first time:</summary>

- [ ] Install `cargo-release` if needed: `cargo install cargo-release`
- [ ] Check that you are logged into `crates.io` as an owner of the crate or [a member of the `owners` group](https://github.com/orgs/ZcashFoundation/teams/owners)

</details>

- [ ] Do a dry-run:
    - [ ] `cargo release --verbose --workspace --exclude zebrad beta`
    - [ ] `cargo release --verbose --workspace --package zebrad [ major | minor | patch ]`
- [ ] Commit the version changes using `git` and submit them as a Zebra PR
```

So we can delete the `Version Locations` section, and remove the toml changes in `Version Tooling`:
```
fastmod --fixed-strings '1.0.0' '1.1.0' README.md zebra-network/src/constants.rs book/src/user/docker.md
```

I think these steps need to go after the release tagging:
```
- [ ] Run `cargo login`
- [ ] Publish the crates in tree-order: `cargo release --verbose --workspace --execute`
```

Then we can open tickets to automate the rest of the replacements:
https://github.com/crate-ci/cargo-release/blob/master/docs/faq.md#how-do-i-update-my-readme-or-other-files

And maybe automate the release tag (but this seems more complicated than using GitHub):
https://github.com/crate-ci/cargo-release/blob/master/docs/faq.md#how-do-i-apply-a-setting-only-to-one-crate-in-my-workspace
https://github.com/crate-ci/cargo-release/blob/master/docs/faq.md#how-do-i-customize-my-tagging-in-a-workspace
`git push --tags <name>`

Reply:
I tried to test these instructions in PR #6908, but I might not have used these exact commands.

Reply:
I'm not sure what you thought of my other suggestions, here they are as a PR, along with some other cleanups: #6931

Reply:
Here's the bit I added in that PR:

> I think the release steps should go after the fastmod steps, and replace most of them. (We can automate the rest later.)

Reply:
#6931 merged into this branch